lib/components/narrative/line-itin/realtime-time-column.js

```diff
@@ -62,4 +64,13 @@ RealtimeTimeColumn.defaultProps = {
   timeOptions: null
 }
 
-export default RealtimeTimeColumn
+const mapStateToProps = (state, ownProps) => {
+  return {
+    homeTimezone: state.otp.config.homeTimezone
+  }
+}
+
+const mapDispatchToProps = {
+}
```

Comment:
Delete this var, and delete it from the `connect` function's arguments.

Reply:
Removed in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/898f5695ac99f2f781d768933da78cbbada87161